### PR TITLE
feat(tokens): provide autofix for all global tokens

### PR DESCRIPTION
**Pull request by adamviktora**
Closes https://github.com/patternfly/pf-codemods/issues/748

Provides an autofix for both the non-color and color global tokens.

There could be a problem when importing the `temp_dev_tbd` token, if consumers are using an old version of patternfly - it has been added recently in one of the newest post beta versions.

### CODE REVIEW

**Review by wise-king-sullyman — APPROVED** [review]
Two nits, and I wonder if we should change this rule name since it's no longer a warning, but nothing I'd block over.

File: packages/eslint-plugin-pf-codemods/src/rules/v6/tokensWarn/tokens-warn.ts [hunk 1]
```diff
@@ -1,82 +1,204 @@
...
+            varName as keyof typeof globalNonColorCssVarNamesMap
+          ] !== "SKIP";
+
+        if (shouldReplaceVar) {
+          const newVarName =
+            globalNonColorCssVarNamesMap[
+              varName as keyof typeof globalNonColorCssVarNamesMap
+            ];
+
+          if (newVarName !== "SKIP") {
```
[the remark this answers is not in the document]
Comment (adamviktora) [reply packages/eslint-plugin-pf-codemods/src/rules/v6/tokensWarn/tokens-warn.ts:175]: Good catch, I forgot to delete this if

File: packages/eslint-plugin-pf-codemods/src/rules/v6/tokensUpdate/tokens-update.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+### tokens-update
+
+We have updated our CSS tokens. About half of our tokens have been replaced with newer ones. 
+
+- this rule provides an autofix for global non color tokens
+- global color tokens will be replaced with a temporary hot pink color token (`t_temp_dev_tbd` react token or `--pf-t--temp--dev--tbd` class)
+  - don't forget to replace these tokens later
```
Comment (thatblindgeye) [inline packages/eslint-plugin-pf-codemods/src/rules/v6/tokensUpdate/tokens-update.md:7]:
```suggestion
- global color tokens will be replaced with a temporary hot pink color token **that must be manually replaced** (`t_temp_dev_tbd` react token or `--pf-t--temp--dev--tbd` CSS variable)
```

Slight tweak to make needing to replace this token stand out a little more

**Review by thatblindgeye — APPROVED** [review]

**Review by kmcfaul — APPROVED** [review]

**Review by wise-king-sullyman — APPROVED** [review]
🔥 🔥 🔥 absolutely AWESOME work on this!

**Pull request merged**